### PR TITLE
Creating class content schemas

### PR DESCRIPTION
**Pull request by sbilge**
It involves the changes to extract the class relations and to create class content JSON schemas using the main schema.

### CODE REVIEW

**Review by KerstenBreuer — CHANGES_REQUESTED** [review]
Looks good, just a few note. For understanding, this is targeting step iv of the procedure described in the tech spec: https://github.com/ghga-de/epic-docs/blob/main/49-jackal/technical_specification.md#i-schemapack-based-re-implementation-of-the-metadata-model

File: src/schemas/File.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        "checksum_type": {
+            "description": "The type of algorithm used to generate the checksum of the File.",
+            "type": "string"
+        },
+        "format": {
+            "$ref": "/workspace/src/schemas/FileFormatEnum",
+            "description": "The format of the File (e.g., BAM, SAM, CRAM, BAI)."
+        },
+        "forward_or_reverse": {
+            "$ref": "/workspace/src/schemas/ForwardOrReverseEnum",
```
[the remark this answers is not in the document]
Comment (sbilge) [reply src/schemas/File.json]: Yes, that is what I plan to

File: exclude_config.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- AliasMixin
+- IdentifiedByAliasMixin
+- AttributeMixin
+- Submission
```
Comment (KerstenBreuer) [inline exclude_config.yaml:4]: Maybe add newline, could also be done automatically using pre-commit

**Review by KerstenBreuer — APPROVED** [review]
Thanks for addressing the points

**Pull request merged**